src/functions.js

```diff
@@ -513,6 +513,28 @@ var _ = Mavo.Functions = {
 
 var $u = _.util;
 
+/**
+ * After plugins are loaded, enable
+ * multi-valued arguments of Mavo and Math functions
+ */
+requestAnimationFrame(() => {
+	Mavo.dependencies[0].then(() => {
+		Object.getOwnPropertyNames(Mavo.Functions).forEach(property => {
+			const FN = Mavo.Functions[property];
+			const ARRAYS = FN.multiValued;
```

Comment:
It would be useful to enable this on some functions, in the same PR. Otherwise once this is merged, it just adds extra code with no immediate benefit.

- Are there any functions that can be simplified with this? I.e. functions that currently take care of arrays manually and we can shorten them by just setting this flag on them?
- There are two ways to enable this flag on the functions that need it:
	- In the function definition, changing `functionName: function(){...}` to `functionName: $.extend(function(){...}, {multiValued: true})`. This may get a little repetitive for many functions, but is very explicit (an the only solution for values other than `true`). I also think it might be *more* DRY than other solutions, despite the repetitiveness, as we don't need to maintain info about functions in two places.
	- After all functions are defined, in a loop that goes over a list of names and setting `multiValued` to `true`. This may be good for handling them in bulk, but we do need to maintain names in two places (and change them in two places) and we still need to use the first way for values other than `true`.

Reply:
> Are there any functions that can be simplified with this? I.e. functions that currently take care of arrays manually and we can shorten them by just setting this flag on them?

There are quite a lot of functions that call `binaryOperation()` so we could definitely simply them.

> There are two ways to enable this flag on the functions that need it:

I do like the first way more than the latter, and I believe it's what we originally came up with.